src/app/pages/fund/components/fiat-providers-list.tsx

```diff
@@ -43,7 +43,12 @@ export const FiatProvidersList = (props: FiatProvidersProps) => {
     >
       <ReceiveStxItem onReceiveStx={() => navigate(RouteUrls.FundReceive)} />
       {Object.entries(activeProviders).map(([providerKey, providerValue]) => {
-        const providerUrl = getProviderUrl(address, providerKey, providerValue.name);
+        const providerUrl = getProviderUrl({
+          address,
+          providerHasFastCheckoutProcess: providerValue.hasFastCheckoutProcess,
+          providerKey,
+          providerName: providerValue.name,
```

Comment:
Very minor remark, but could probably omit "provider" from key names on grounds of over qualification (described in method name)